buildroot-external/rootfs-overlay/usr/sbin/hassos-cli

```diff
@@ -19,7 +19,17 @@ docker container exec \
     -ti hassio_cli \
     /usr/bin/cli.sh
 
-# Jump to root login shell
-if [ $? -eq 10 ]; then
+case $? in
+  10)
+    # Jump to root login shell (login command)
     exec /bin/ash -l
-fi
+    ;;
+  143)
+    # 143 graceful termination (SIGTERM). Most likely a proper shutdown.
+    # Just sleep for a while until actual systemd shutdown gets invoked.
+    sleep 15
```

Comment:
Ideally we should write something, otherwise the CLI just looks "dead" during this sleep.

```suggestion
    echo ""
    echo "Home Assistant CLI has been terminated."
    sleep 15
```